NetTopologySuite/Index/KdTree/KdTreeExtensions.cs

```diff
@@ -77,8 +77,11 @@ public static class KdTreeExtensions
             if (isOddLevel)
                 return (left ? target.X <= node.X : target.X >= node.X) || Math.Pow(target.X - node.X, 2) < closestDistSq;
             return (left ? target.Y <= node.Y : target.Y >= node.Y) || Math.Pow(target.Y - node.Y, 2) < closestDistSq;
+
+/*
```

Comment:
Reverted this.  Looks like I'd taken it from [here](https://github.com/NetTopologySuite/NetTopologySuite/commit/76cc663f430b9323475c7592d7b79d0cb91930f6#diff-0df2957c80cc061d734fc162aa7a2781), and I approved of this particular change (commenting out provably dead code) even though this file no longer otherwise **needs** to be changed.